gapic/src/main/com/google/gapid/models/Settings.java

```diff
@@ -88,6 +88,7 @@
   public String[] recentFiles = new String[0];
   public String adb = "";
   public boolean autoCheckForUpdates = true;
+  public boolean autoCheckForUpdatePrereleases = false;
```

Comment:
autoCheckForUpdatePrereleases -> onDevTrack
updates.autoCheckPrerelease -> updates.devTrack

Reply:
I renamed to "includePrerelease" to keep the "prerelease" naming because:
- it reflects the GitHub naming
- it is already used in other parts of GAPIC, e.g. in function `checkForUpdates(boolean includePrereleases)`